### ISSUE TITLE
A error comes while I run the code.

### ISSUE THREAD

**Issue by ghost** [norishigefukushima/EfficientLargeScaleStereo#2]
the following is where the error happen:

  for (i = 0; i < m->invertices; i++) {
    vertexloop = (vertex) poolalloc(&m->vertices);
    /* Read the vertex coordinates. */
//*******************ERROR****************
	x = vertexloop[0] = pointlist[coordindex++];
	y = vertexloop[1] = pointlist[coordindex++];
//*******************ERROR****************
    /* Read the vertex attributes. */
    for (j = 0; j < numberofpointattribs; j++) {
      vertexloop[2 + j] = pointattriblist[attribindex++];
    }
    if (pointmarkerlist != (int *) NULL) {
      /* Read a vertex marker. */
      setvertexmark(vertexloop, pointmarkerlist[i]);
    } else {
      /* If no markers are specified, they default to zero. */
      setvertexmark(vertexloop, 0);
    }
    setvertextype(vertexloop, INPUTVERTEX);
    /* Determine the smallest and largest x and y coordinates. */
    if (i == 0) {
      m->xmin = m->xmax = x;
      m->ymin = m->ymax = y;
    } else {
      m->xmin = (x < m->xmin) ? x : m->xmin;
      m->xmax = (x > m->xmax) ? x : m->xmax;
      m->ymin = (y < m->ymin) ? y : m->ymin;
      m->ymax = (y > m->ymax) ? y : m->ymax;
    }
  }
I hope you can help me fix the problem, thank you very much.

**Comment by zdluffy** [comment]
I got the same problem. Have you solved this problem?

**Comment by ghost** [comment]
> I got the same problem. Have you solved this problem?

No, I did not get the answer, and I got no answer from anyone. If you want fix this problem,  you should read the article.

**Comment by zdluffy** [comment]
ok,thank you for reply.------------------ Original ------------------

**Comment by jjhlc** [comment]
I got the same problem ,Did you know the reason?

**Comment by diriki** [comment]
I got the same problem.
I solved replacing files of libelas (except OpenCV wrapper class) with their updated version from http://www.cvlibs.net/software/libelas
